README.md

```diff
@@ -67,6 +67,14 @@ Change the listening port of the gRPC server. Default is `0` and will automatica
 GF_RENDERER_PLUGIN_GRPC_PORT=50059
 ```
 
+**Verbose logging:**
+
+Enable capturing verbose information when headless Chrome browses a page and takes a screenshot of it. It will capture all requests initiated and finished, when page is closed and any non-error logged to the console. These are logged as debug messages. Default is `false` and will only capture  errors when page crashes, uncaught exception thrown in page, page request fails or error logged to the console.
```

Comment:
I am not sure what you are trying to say. Is this right?

Verbose logging allows you to capture verbose information by taking a screenshot when headless Chrome browses a page. You can set it to `true` or `false`.

* `false`: Default state. Logging only captures errors when a page crashes, uncaught exception are thrown in a page, a page request fails, or an error is logged to the console.
* `true`: Captures all information as `false`, as well as requests initiated and finished, when a page is closed, and any non-error logged to the console.

Reply:
> Verbose logging allows you to capture verbose information by taking a screenshot when headless Chrome browses a page. You can set it to true or false.

Not really, so basically a browser are run in headless mode, navigating to requested link/dashboard panel/dashboard and waits for the data to be loaded then takes a screenshot. What I'm trying to describe is that you can enable capture of verbose information during this process. Does that makes sense to you?

Splitting up the different values in bullet-list is a good suggestion. Thanks

Reply:
So that active explanation that you just gave me of what the browser does was much easier to understand than the passive-voice explanation in the text. I recommend rewriting the document to match it.

When a browser runs in headless mode, navigates to the requested link, waits for data to be loaded, and then takes a screenshot of the data. You can tell Grafana to capture verbose information during the data (true), or you can leave the settings at default levels (false).

Reply:
Tried to simplify it. What do you think?